### DISCUSSION TITLE
An error occured when changing application projects

### DISCUSSION THREAD

**Discussion by ep4sh** [argoproj/argo-cd#13700] [General]
I'm performing housekeeping in ArgoCD and assigning proper projects for each app.
I've found some apps I can't reassign from the default project:
```
elasticsearch-curator
gitlab
gitlab-runner-cd
nexus
....
```

the error looks like the following:
![image](https://github.com/argoproj/argo-cd/assets/19505042/ff07064e-79e9-4e0a-a5d1-98cecd9d61a9)

Does anyone know what's the issue?

**Comment by mubarak-j** [comment]
have you tried [changing the project](https://argo-cd.readthedocs.io/en/stable/user-guide/projects/#assign-application-to-a-project) via CLI?
`argocd app set guestbook-default --project <new_project>`

if that doesn't work, can you share the output of the following?
`kubectl get appproject -n argocd <new_project> -o yaml`

there might be some default restrictions on the new project you're trying to move to. Also take a look at the [RBAC rules in the docs](https://argo-cd.readthedocs.io/en/stable/user-guide/projects/#configuring-rbac-with-projects).

**Reply by ep4sh** [reply]
Thank you, that helped!
